EU4toV2/Source/V2World/V2Country.h

```diff
@@ -129,7 +131,7 @@ class V2Country
 
 		string							getLocalName();
 		std::optional<V2Relations> getRelations(std::string withWhom) const;
-		void getNationalValueScores(int& liberty, int& equality, int& order, const mappers::IdeaEffectMapper& ideaEffectMapper);
+		void getNationalValueScores(double& liberty, double& equality, double& order);
```

Comment:
Since you're modifying this function, it should return a structure (best) or std::tuple (okay) instead of modifying parameters.

Reply:
tuple it is. these values are only used here and nowhere else and a struct seems unnecessary.